Berksfile

```diff
@@ -0,0 +1,3 @@
+source "https://supermarket.getchef.com"
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.